ols/src/llms/requirements.txt

```diff
@@ -1,4 +1,4 @@
 openai
 ibm_watson_machine_learning
-ibm-generative-ai
+ibm-generative-ai==0.6.1
```

Comment:
why is it needed for unit tests?

Reply:
because the latest version of `ibm-generative-ai` is 2.0.0 that breaks our code on [an import of genai.schema](https://github.com/openshift/lightspeed-service/blob/2a458752ec0908e32480ba0935c469b5c3db8c9b/ols/src/llms/llm_loader.py#L153)
The v2.0.0 has removed that module of genai.schema.

Reply:
yup, but you can mock the LLM loader completely